mono/mini/mini-runtime.c

```diff
@@ -2593,6 +2595,33 @@ mono_jit_free_method (MonoDomain *domain, MonoMethod *method)
 	g_free (ji);
 }
 
+gpointer
+mono_jit_search_all_backends_for_jit_info (MonoDomain *domain, MonoMethod *method, MonoJitInfo **out_ji)
+{
+	gpointer code;
+	MonoJitInfo *ji;
+
+	code = mono_jit_find_compiled_method_with_jit_info (domain, method, &ji);
+	if (!code) {
+		ERROR_DECL_VALUE (oerror);
+
+		/* Might be AOTed code */
+		mono_class_init (method->klass);
+		code = mono_aot_get_method (domain, method, &oerror);
+		if (code) {
+			mono_error_assert_ok (&oerror);
+			ji = mono_jit_info_table_find (domain, code);
+		} else {
+			/* Might be interpreted */
```

Comment:
this branch leaks `oerror` if it's not ok.  need to `mono_error_cleanup (&oerror)`

Reply:
I have added the related cleanup